src/client/conn.rs

```diff
@@ -7,6 +7,41 @@
 //!
 //! If don't have need to manage connections yourself, consider using the
 //! higher-level [Client](super) API.
+//!
+//! ## Example
+//! A simple example that uses the `SendRequest` struct to talk HTTP over a Tokio TCP stream
+//! ```no_run
+//! # #[cfg(all(feature = "client", feature = "http1", feature = "runtime"))]
+//! # mod rt {
+//! use http::{Request, StatusCode};
+//! use hyper::{client::conn::Builder, Body};
+//! use tokio::net::TcpStream;
+//!
+//! #[tokio::main]
+//! async fn main() {
```

Comment:
Most of the docs follow the style of using `?` instead of `.unwrap()`, so I'd change this to `async fn main() -> Result<(), Box<dyn std::error::Error>> {` (and replace all the `unwrap`s with `?`).